Mixpanel/MPNotificationViewController.m

```diff
@@ -106,14 +106,16 @@ - (void)viewDidLoad {
 
         MPTakeoverNotification *notification = (MPTakeoverNotification *) self.notification;
 
-        if (notification.title && notification.body) {
+        if ([notification.title isEqual:[NSNull null]] ||
```

Comment:
this is pretty messy and there's no reason to handle nsnull here - can we do that at parse time and only check for nil in the view controllers?